src/test/java/teammates/test/cases/browsertests/FeedbackRankQuestionUiTest.java

```diff
@@ -176,35 +176,35 @@ public void testInstructorSubmitAndResultsPage() throws Exception {
 
         InstructorFeedbackResultsPage instructorResultsPage =
                 loginToInstructorFeedbackResultsPageWithViewType("instructor1", "instructor", false, "question");
-        clickAjaxLoadedPanelAndWaitForExpansion(instructorResultsPage, "panelHeading-1", "ajax_auto");
+        instructorResultsPage.clickAjaxLoadedPanelAndWaitForExpansion("panelHeading-1", "ajax_auto");
```

Comment:
Looks like `ajax-auto` is always passed as a parameter. Can we put that inside the method itself?
Can the method name be higher level e.g. `loadPanel("panelHeading-1")`. Even better if it can be something like `loadResultPanel(1)` because `"panelHeading-1"` is an implementation detail of the page that should not be leaked through the API. The API of *Page classes should speak the language of the user (because the test is simulating a user). A user can say thing like `load results of the first question`, not things like `load the panel that uses ajax and has the id xyz and class efg`.